README.md

```diff
@@ -90,7 +92,7 @@ Which of the following represents a correct workflow for submitting a PR on a no
 Select 1:
 ```
 [] fork on github; git clone <fork_url>; git checkout -b <charlie_solution>; git add <files>; git commit; git push; create pull request
-[] fork on github; git clone <ga_dc_url>; git checkout -b <charlie_solution>; git add <files>; git commit; git push; create pull request
+[X] fork on github; git clone <ga_dc_url>; git checkout -b <charlie_solution>; git add <files>; git commit; git push; create pull request
```

Comment:
Check the solution. You don't want to clone GA's version of the URL because you don't have permission to push changes up to it.